### PR TITLE
Allow for installing custom fonts in office preview

### PR DESCRIPTION
**Pull request by colszowka**
https://www.pivotaltracker.com/n/projects/1489738

Will restart unoconv listener when mounted custom fonts change

### CODE REVIEW

File: services/init-protonet.service [hunk 1]
```diff
@@ -33,6 +34,7 @@ ExecStartPre=/bin/chown 1000:1000 /data/soul/log
 ExecStartPre=/bin/chown 1000:1000 /data/soul/public
 ExecStartPre=/bin/chown 1000:1000 /data/soul/uploads
 ExecStartPre=/bin/chown 1000:1000 /data/soul/protosync
+ExecStartPre=/bin/chown 1000:1000 /data/fonts
```
Comment (hinnerk) [inline services/init-protonet.service]: Since we use a uid specific to soul I'd put this under `/data/soul/fonts` (or `/data/unoconv/fonts`).

Comment (colszowka) [reply services/init-protonet.service]: The docker-image on the other hand is called `platform-unoconv`, is public, and at least in theory could be used by other clients, hence I went for the top level variant

**Comment by colszowka** [comment]
@hinnerk As discussed in person, adjusted the path for the fonts to `/data/unoconv/fonts`

**Comment by colszowka** [comment]
Adjusted docs PR accordingly https://github.com/protonet/documentation/pull/25

**Pull request merged**